kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/ExecWebSocketListener.java

```diff
@@ -196,7 +196,7 @@ public void onFailure(WebSocket webSocket, Throwable t, Response response) {
       if (explicitlyClosed.get() || closed.get() || !failed.compareAndSet(false, true) ) {
         //We are not going to notify the listener, sicne we've already called onClose(), so let's log a debug/warning.
         if (LOGGER.isDebugEnabled()) {
-          LOGGER.warn("Received [" + t.getClass().getCanonicalName() + "], with message:[" + t.getMessage() + "] after ExecWebSocketListener is closed, Ignoring.");
+          LOGGER.warn("Received [%s], with message:[%s] after ExecWebSocketListener is closed, Ignoring.",t.getClass().getCanonicalName(),t.getMessage());
```

Comment:
I'm not sure slf4j works with `%s`, Shouldn't this be `{}` here instead?
```suggestion
          LOGGER.warn("Received [{}], with message:[{}] after ExecWebSocketListener is closed, Ignoring.",t.getClass().getCanonicalName(),t.getMessage());
```

Reply:
yeah, I had the same doubt, that's why I asked, thanks for the suggestion.